src/vma/sock/sockinfo_udp.cpp

```diff
@@ -1830,6 +1830,12 @@ bool sockinfo_udp::rx_input_cb(mem_buf_desc_t* p_desc, void* pv_fd_ready_array)
 		}
 	}
 
+	if ((m_b_rcvtstamp ||
+		 (m_n_tsing_flags &
+		  (SOF_TIMESTAMPING_RX_SOFTWARE | SOF_TIMESTAMPING_SOFTWARE))) &&
+		!p_desc->path.rx.sw_timestamp) {
```

Comment:
@rafiw what is the meaning of ! operation on a struct?
Just tried compiling and it fails!!!